slsDetectorSoftware/src/Module.cpp

```diff
@@ -2105,22 +2064,16 @@ void Module::setROI(slsDetectorDefs::ROI arg) {
         arg.xmin = -1;
         arg.xmax = -1;
     }
-    std::array<int, 2> args{arg.xmin, arg.xmax};
     LOG(logDEBUG) << "Sending ROI to detector [" << arg.xmin << ", " << arg.xmax
                   << "]";
-    sendToDetector(F_SET_ROI, args, nullptr);
+    sendToDetector(F_SET_ROI, arg, nullptr);
```

Comment:
can we send the ROI structure to the detector? being 64 and 32 bit is okay?

Reply:
Struct is declared with attribute packed, should be the same as sending two ints.

Reply:
Alright! :+1: